ElementX/Sources/Application/AppSettings.swift

```diff
@@ -46,6 +46,7 @@ final class AppSettings {
         case fuzzyRoomListSearchEnabled
         case pinningEnabled
         case enableOnlySignedDeviceIsolationMode
+        case identityPinningViolationNotifications
```

Comment:
Typically our keys tend to match the property name I think?
```suggestion
        case identityPinningViolationNotificationsEnabled
```